www/install/templates/step1.tpl

```diff
@@ -1,249 +1,251 @@
-	<td>{if $cfg->PDOCheck}<span class="success">OK</span>{else}<span class="error">Error</span>{/if}</td>
-</tr>
-<tr class="">
-	<td>Checking PHP GD extension:{if !$cfg->gdCheck}<br /><span class="warn">The PHP installation lacks support for GD.</span>{/if}</td>
-	<td>{if $cfg->gdCheck}<span class="success">OK</span>{else}<span class="warn">Warning</span>{/if}</td>
-</tr>
-<tr class="alt">
-	<td>Checking PHP OpenSSL Extension:{if !$cfg->opensslCheck}<br /><span class="warn">Your PHP installation does not have the openssl extension loaded. SSL Usenet connections will fail.</span>{/if}</td>
-	<td>{if $cfg->opensslCheck}<span class="success">OK</span>{else}<span class="warn">Warning</span>{/if}</td>
-</tr>
-<tr class="">
-	<td>Checking PHP EXIF Extension:{if !$cfg->exifCheck}<br /><span class="warn">Your PHP installation does not have the exif extension loaded. It is required for NFO checking and various other functions.</span>{/if}</td>
-	<td>{if $cfg->exifCheck}<span class="success">OK</span>{else}<span class="warn">Warning</span>{/if}</td>
-</tr>
-<tr class="alt">
-	<td>Checking PHP json extension:{if !$cfg->jsonCheck}<br /><span class="error">The PHP installation lacks support for json.</span>{/if}</td>
-	<td>{if $cfg->jsonCheck}<span class="success">OK</span>{else}<span class="error">Error</span>{/if}</td>
-</tr>
-<tr class="">
-	<td>Checking PHP date.timezone:{if !$cfg->timezoneCheck}<br /><span class="warn">You have no default timezone set in php.ini. e.g date.timezone = America/New_York</span>{/if}</td>
-	<td>{if $cfg->timezoneCheck}<span class="success">OK</span>{else}<span class="warn">Warning</span>{/if}</td>
-</tr>
-<tr class="alt">
-	<td>Checking PHP max_execution_time:{if !$cfg->timelimitCheck}<br /><span class="warn">Your PHP installation's max_execution_time setting is low, please consider increasing it >= 120</span>{/if}</td>
-	<td>{if $cfg->timelimitCheck}<span class="success">OK</span>{else}<span class="warn">Warning</span>{/if}</td>
-</tr>
-<tr class="">
-	<td>Checking PHP memory_limit:{if !$cfg->memlimitCheck}<br /><span class="warn">Your PHP installation's memory_limit setting is low, please consider increasing it >= 1024M.</span>{/if}</td>
-	<td>{if $cfg->memlimitCheck}<span class="success">OK</span>{else}<span class="warn">Warning</span>{/if}</td>
-</tr>
-<tr class="alt">
-	<td>Checking if the Smarty compile dir is writable:
-		{if !$cfg->cacheCheck}<br />
-			<span class="error">
-					The template compile dir must be writable.<br /><br />
-					A quick solution is to run:<br />
-					chmod 777 {$cfg->SMARTY_DIR}/templates_c
-				{$fixString}
-				</span>
+	<tr>
+		<th>check</th>
+		<th style="width:75px;">status</th>
+	</tr>
+	<tr class="alt">
+		<td>Checking for iconv support:{if !$cfg->iconvCheck}<br /><span class="error">The PHP installation lacks support for iconv.</span>{/if}</td>
+		<td>{if $cfg->iconvCheck}<span class="success">OK</span>{else}<span class="error">Error</span>{/if}</td>
+	</tr>
```

Comment:
Line 5 to 9 are the only real changes, the rest was an indentation fix.

Indentation fix: https://github.com/kevinlekiller/nZEDb/commit/694cc37e172be5d06e5270f53fde4f5969c3905f

Line 5 to 9 addition: https://github.com/kevinlekiller/nZEDb/commit/fe4017932225871f1115dd017376617756c9d644#diff-8dc357d49a7587abf80090cc2bc9c390L5